### PR TITLE
[doc] Doc cleanup and updates for 1.1.0 release

### PR DESCRIPTION
**Pull request by gyfora**
## What is the purpose of the change

Doc updates for the upcoming release

### CODE REVIEW

**Comment by gyfora** [comment]
cc @morhidi

File: docs/content/docs/operations/metrics-logging.md [hunk 1]
```diff
@@ -24,33 +24,26 @@ specific language governing permissions and limitations
...
+### Flink Resource Metrics
+The Operator gathers aggregates metrics about managed resources.
 
-### Lifecycle metrics
+| Scope              | Metrics                                                                                   | Description                                                                                                                                                                                        | Type      |
+|--------------------|-------------------------------------------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|-----------|
+| Namespace          | FlinkDeployment/FlinkSessionJob.Count                                                     | Number of managed FlinkDeployment/SessionJob instances per namespace                                                                                                                               | Gauge     |
```
Comment (morhidi) [inline docs/content/docs/operations/metrics-logging.md:38]: nit: FlinkDeployment/SessionJob->FlinkDeployment/FlinkSessionJob

File: docs/content/docs/operations/metrics-logging.md [hunk 2]
```diff
@@ -63,7 +56,7 @@ In addition to the simple counts we further track a few selected state transitio
  - Rollback : Time from deployed to rolled_back state if the resource was rolled back
  - Submission: Flink resource submission time
 
-## Kubernetes Client Metrics
+### Kubernetes Client Metrics
 
 The Operator gathers various metrics related to Kubernetes API server access. The Kubernetes client metrics can be enabled by the configuration `kubernetes.operator.kubernetes.client.metrics.enabled` (default: `true`).
```
Comment (morhidi) [inline docs/content/docs/operations/metrics-logging.md:61]: We could probably remove the property from here. (or add it everywhere :)

**Comment by morhidi** [comment]
+1 LGTM

File: docs/content/docs/operations/metrics-logging.md (hunk 1 above)
Comment (gyfora) [reply docs/content/docs/operations/metrics-logging.md]: fixed

File: docs/content/docs/operations/metrics-logging.md (hunk 2 above)
Comment (gyfora) [reply docs/content/docs/operations/metrics-logging.md]: I added a general note and a link to the general config section

**Pull request merged**